### PR TITLE
fix: autofill and autofill color for inputs

### PR DESCRIPTION
**Pull request by mikaelbr**
Gjør slik at inputs ikke får rar farge ved autoutfylling. Også at engangspassord ikke får autocomplete, da det ikke gir mening.

### CODE REVIEW

File: src/static/styles/ui.scss [hunk 1]
```diff
@@ -490,8 +490,24 @@
     width: 100%;
     padding: 0;
     border: 0;
-    background: transparent;
+    background-color: transparent;
+
+    &:-webkit-autofill,
```
Comment (mjenssen) [inline src/static/styles/ui.scss:495]: Shouldn't this use the `:autofill` pseudo-class and let postcss or whatever convert this to what is needed for the different browsers?

File: src/static/styles/ui.scss [hunk 2]
```diff
@@ -490,8 +490,24 @@
...
+    &:-webkit-autofill:hover,
+    &:-webkit-autofill:focus,
+    &:-webkit-autofill:active {
+        transition: background-color 5000s ease-in-out 0s;
+        -webkit-text-fill-color: inherit !important;
+    }
+}
+@keyframes autofill {
```
Comment (mjenssen) [inline src/static/styles/ui.scss:503]: Where is this used?

File: src/static/styles/ui.scss (hunk 1 above)
Comment (mikaelbr) [reply src/static/styles/ui.scss:495]: Not yet prefixed by autoprefixer: https://github.com/postcss/autoprefixer/issues/626

File: src/static/styles/ui.scss (hunk 2 above)
Comment (mikaelbr) [reply src/static/styles/ui.scss]: Absolutely no-where. Remnants of hackish exploration.

**Review by mjenssen — APPROVED** [review]

**Pull request merged**